WooCommerce/Classes/ServiceLocator/ServiceLocator.swift

```diff
@@ -52,12 +52,16 @@ final class ServiceLocator {
 
     /// CoreData Stack
     ///
-    private static var _storageManager = CoreDataManager(name: WooConstants.databaseStackName, crashLogger: SentryCrashLogger())
+    private static var _storageManager = CoreDataManager(name: WooConstants.databaseStackName, crashLogger: crashLogging)
 
     /// Cocoalumberjack DDLog
     ///
     private static var _fileLogger: Logs = DDFileLogger()
 
+    /// Crash Logging Stack
+    ///
+    private static var _crashLogging: CrashLoggingStack = WooCrashLoggingStack()
```

Comment:
For my curiosity: why not using a `private(set)` to have the getter public but the setter private (instead of using `_crashLogging` here and `var crashLogging { _crashLogging }` on lines 156-161)?
```suggestion
    private(set) static var crashLogging: CrashLoggingStack = WooCrashLoggingStack()
```

Reply:
I was following the pattern of how everything else does this in `ServiceLocator`. I think that might result in a cleaner implementation, but if we go for it we should do it for all the methods in `ServiceLocator`